### ISSUE TITLE
[FEATURE] Manually update animations without `requestAnimationFrame`

### ISSUE THREAD

**Issue by mattrossman** [framer/motion#2496]
**Is your feature request related to a problem? Please describe.**

Framer Motion's animations stop working in scenarios where `window.requestAnimationFrame` isn't available.

See https://github.com/pmndrs/react-xr/issues/180 for an example of how this is an issue when using `framer-motion` or `framer-motion-3d` in a WebXR immersive session.

**Describe the solution you'd like**

Expose some way of manually updating animations, so users can pick an appropriate looping mechanism when requestAnimationFrame is unavailable.

**Describe alternatives you've considered**
n/a

**Additional context**

GSAP provides an `.updateRoot()` method for this purpose: https://gsap.com/docs/v3/GSAP/gsap.updateRoot()/

**Comment by jjrchrds** [comment]
@mattrossman were you able to find another solution for this?

**Comment by mattrossman** [comment]
@jjrchrds No, I ended up using a different animation library (react-spring) due to this limitation.
